src/nb_clean/__init__.py

```diff
@@ -227,6 +227,10 @@ def clean_notebook(
             cell["execution_count"] = None
             if not preserve_cell_outputs:
                 cell["outputs"] = []
+            else:
+                for dict_ in cell["outputs"]:
```

Comment:
Can we call the loop variable `output` instead of `dict_`?

Reply:
Sure